homeassistant/components/synology_dsm/camera.py

```diff
@@ -70,33 +76,31 @@ def __init__(
             },
             coordinator,
         )
-        Camera.__init__(self)
+        Camera.__init__(self)  # type: ignore[no-untyped-call]
 
         self._camera_id = camera_id
-        self._api = api
+        self.coordinator: DataUpdateCoordinator[dict[str, dict[str, SynoCamera]]]
```

Comment:
It should be outside init. See this comment: https://github.com/home-assistant/core/pull/49898#issuecomment-830621906

Reply:
ok, now I'm a little bit confused - why should be the definition outside of init or what is the difference between both solutions?

Reply:
I'm surprised it works inside it init. Having it outside is cleaner and tells type checker that this field of this class has type different from parent class.